markets/perps-market/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@synthetixio/perps-market",
-  "version": "3.3.15",
+  "version": "3.3.16",
```

Not a good idea to do it this way.
If we start changing manually all the versions - we must then remove lerna and start doing this manually. We cannot do both.

```suggestion
  "version": "3.3.15",
```